docs/setup/install/installation-configuration.md

```diff
@@ -1,29 +1,94 @@
+| **DEX\_CONFIG** | dex config if you want to integrate login with SSO \(optional\) for more information check [Argocd documentation](https://argoproj.github.io/argo-cd/operator-manual/user-management/) |  |
+| **EXTERNAL\_SECRET\_AMAZON\_REGION** | AWS region for the secret manager to pick \(required\) |  |
+| **PROMETHEUS\_URL** | URL of Prometheus where all cluster data is stored; if this is wrong, you will not be able to see application metrics like CPU, RAM, HTTP status code, latency, and throughput \(required\) |  |
+
+## Storage for Logs and Cache
+
+### AWS SPECIFIC	
+
+While installing Devtron and using the AWS-S3 bucket for storing the logs and caches, the below parameters are to be used in the ConfigMap.
+
+> NOTE: For using the S3 bucket it is important to add the S3 permission policy to the IAM role attached to the nodes of the cluster.
+
+| Parameter | Description | Default |
+| :--- | :--- | :--- |
+| **DEFAULT\_CACHE\_BUCKET** | AWS bucket to store docker cache, it should be created beforehand \(required\) |  |
+| **DEFAULT\_BUILD\_LOGS\_BUCKET** | AWS bucket to store build logs, it should be created beforehand \(required\) |  |
+| **DEFAULT\_CACHE\_BUCKET\_REGION** | AWS region of S3 bucket to store cache \(required\) |  |
+| **DEFAULT\_CD\_LOGS\_BUCKET\_REGION** | AWS region of S3 bucket to store CD logs \(required\) |  |
+
+### AZURE SPECIFIC
+
+While installing Devtron using Azure Blob Storage for storing logs and caches, the below parameters will be used in the ConfigMap.
+
+| Parameter | Description | Default |
+| :--- | :--- | :--- |
+| **AZURE\_ACCOUNT\_NAME** | Account name for AZURE Blob Storage |  |
+| **AZURE\_BLOB\_CONTAINER\_CI\_LOG** | AZURE Blob storage container for storing ci-logs after running the CI pipeline |  |
+| **AZURE\_BLOB\_CONTAINER\_CI\_CACHE** | AZURE Blob storage container for storing ci cache after running the CI pipeline |  |
+
+To convert string to base64 use the following command:
+
+```bash
+echo -n "string" | base64 -d
+```
+
+> **Note**:
+> 1. Ensure that the **cluster has read and write access** to the S3 buckets/Azure Blob storage container mentioned in DEFAULT\_CACHE\_BUCKET, DEFAULT\_BUILD\_LOGS\_BUCKET or AZURE\_BLOB\_CONTAINER\_CI\_LOG, or AZURE\_BLOB\_CONTAINER\_CI\_CACHE.
+> 2. Ensure that the cluster has **read access** to AWS secrets backends \(SSM & secrets manager\).
+
+---
+
+The following tables contain parameters and their details for Secrets and ConfigMaps that are configured during the installation of Devtron. 
+While installing Devtron using `kubectl` the following parameters can be tweaked in [devtron-operator-configs.yaml](https://github.com/devtron-labs/devtron/blob/main/manifests/install/devtron-operator-configs.yaml) file. If the installation is proceeded using `helm3`, the values can be tweaked in [values.yaml](https://github.com/devtron-labs/charts/blob/main/charts/devtron/values.yaml) file.
```

Comment:
We don't have an option to install using kubectl so we can remove that info from here